core/core.go

```diff
@@ -129,54 +140,75 @@ func (c *Core) InsertChain(blocks types.Blocks) (int, error) {
 func (c *Core) procAppendQueue() {
 	// Sort the blocks by number and attempt to insert them
```

Comment:
They are no longer simply sorted by number. Please update this comment

Reply:
updated the comments to reflect the updated logic